mlflow/models/model.py

```diff
@@ -1114,8 +1114,7 @@ def log_model_metrics_for_step(client, model_id, run_id, step):
                     await_registration_for=await_registration_for,
                     local_model_path=local_path,
                 )
-
-            model_info = mlflow_model.get_model_info()
+            model_info = mlflow_model.get_model_info(model)
```

Comment:
Why do we pass model?

Reply:
Good question. To extract some attributes to `ModelInfo`.